.github/workflows/test-unit.yml

```diff
@@ -18,22 +18,27 @@ jobs:
       - uses: actions/checkout@v3
         with: { fetch-depth: 1 }
       - uses: actions/setup-go@v3
-        with: { go-version: "${{ matrix.go }}" }
+        with:
+          go-version: "${{ matrix.go }}"
       - run: go version
       - run: make test-unit race=true
-      - uses: codecov/codecov-action@v1
+      - name: Upload coverage report
+        uses: codecov/codecov-action@v1
         with:
-          file: tmp/unit.cov
-          flags: unit
-        if: matrix.os == 'ubuntu-latest'
-
+          file: coverage.out
+      - name: Comment coverage report
+        run: |
+          if [ "${{ matrix.os }}" == "ubuntu-latest" ] || [ "${{ matrix.os }}" == "windows-latest" ] || [ "${{ matrix.os }}" == "macOS-latest" ]; then
+          curl -sSfL https://codecov.io/bash | bash -s -- -Z -K -f coverage.out || echo "Codecov upload failed"
+          fi
```

Comment:
Feels like this should be the uploader as mentioned in https://about.codecov.io/blog/getting-started-with-code-coverage-for-golang/.

Reply:
Ok I will look into that.